### DISCUSSION TITLE
resend invite to collaborator

### DISCUSSION THREAD

**Discussion by postman31** [community/community#40572] [New to GitHub]
How can I re-send an invite to a collaborator in my private repo? 
I sent an invite some time ago but it wasn't accepted so now it's listed as expired. 
I can't find any other option than 'remove'

**Comment by doguedogue** [comment]
yes, if the invitation expires you have to delete it and resend it.

In the teams there is an option to retry the invitation but here there isn't.

**Comment by mohamedzhioua** [comment]
If an invitee does not accept the invitation within seven days, the pending invitation expires automatically
so you have to delete it and send it again
